addon/components/dfp-ad.js

```diff
@@ -102,6 +102,11 @@ export default Component.extend({
       let target = `ad_${guidFor(this)}`;
       this.set('target', target);
     }
+    this.adState = {
+      isEmpty: true,
+      width: 0,
+      height: 0,
+    }
```

Comment:
this might cause unexpected behavior because it's not using a setter, so templates and computeds won't be updated, but maybe that's fine. if you're seeing unexpected behavior, update this to use `this.setProperties`

Reply:
not entirely sure whether this matters in the `init` step, but i'll update it just to be safe